db/migrate/20150729041232_add_piece_type_to_pieces.rb

```diff
@@ -0,0 +1,6 @@
+class AddPieceTypeToPieces < ActiveRecord::Migration
+
+  def change
+  	rename_column :pieces, :type, :piece_type
```

Comment:
I think be default, Rail's expects the STI column to be called `type` :smile: 

If you want, you could have the type column be called `piece_type`. But you'd need to add this line to your Piece model

```Ruby
self.inheritance_column = :piece_type 
```

(Totally possible I may be wrong, haven't done STI in a while)

some more info here: http://samurails.com/tutorial/single-table-inheritance-with-rails-4-part-1/